sdk/src/main/scala/com/horizen/account/AccountSidechainNodeViewHolder.scala

```diff
@@ -0,0 +1,281 @@
+import com.horizen.account.state.{AccountState, MessageProcessor, MessageProcessorUtil}
+import com.horizen.account.storage.{AccountHistoryStorage, AccountStateMetadataStorage}
+import com.horizen.account.transaction.AccountTransaction
+import com.horizen.account.validation.{BaseFeeBlockValidator, ChainIdBlockSemanticValidator}
+import com.horizen.account.wallet.AccountWallet
+import com.horizen.consensus._
+import com.horizen.evm.Database
+import com.horizen.node.NodeWalletBase
+import com.horizen.params.NetworkParams
+import com.horizen.proof.Proof
+import com.horizen.proposition.Proposition
+import com.horizen.storage.{SidechainSecretStorage, SidechainStorageInfo}
+import com.horizen.validation.{HistoryBlockValidator, SemanticBlockValidator}
+import com.horizen.{AbstractSidechainNodeViewHolder, AbstractState, SidechainSettings, SidechainTypes}
+import scorex.util.ModifierId
+import sparkz.core.NodeViewHolder.ReceivableMessages.LocallyGeneratedTransaction
+import sparkz.core.transaction.state.TransactionValidation
+import sparkz.core.utils.NetworkTimeProvider
+import scala.util.Success
+import scala.collection.JavaConverters.collectionAsScalaIterableConverter
+
+class AccountSidechainNodeViewHolder(sidechainSettings: SidechainSettings,
+                                     params: NetworkParams,
+                                     timeProvider: NetworkTimeProvider,
+                                     historyStorage: AccountHistoryStorage,
+                                     consensusDataStorage: ConsensusDataStorage,
+                                     stateMetadataStorage: AccountStateMetadataStorage,
+                                     stateDbStorage: Database,
+                                     customMessageProcessors: Seq[MessageProcessor],
+                                     secretStorage: SidechainSecretStorage,
+                                     genesisBlock: AccountBlock)
+  extends AbstractSidechainNodeViewHolder[SidechainTypes#SCAT, AccountBlockHeader, AccountBlock](sidechainSettings, timeProvider) {
+
+  override type HSTOR = AccountHistoryStorage
+  override type HIS = AccountHistory
+  override type MS = AccountState
+  override type VL = AccountWallet
+  override type MP = AccountMemoryPool
+  override type FPI = AccountFeePaymentsInfo
+
+  protected def messageProcessors(params: NetworkParams): Seq[MessageProcessor] = {
+      MessageProcessorUtil.getMessageProcessorSeq(params, customMessageProcessors)
+  }
+
+  override def semanticBlockValidators(params: NetworkParams): Seq[SemanticBlockValidator[AccountBlock]] = {
+    ChainIdBlockSemanticValidator(params) +: super.semanticBlockValidators(params)
+  }
+
+  override def historyBlockValidators(params: NetworkParams): Seq[HistoryBlockValidator[SidechainTypes#SCAT, AccountBlockHeader, AccountBlock, AccountFeePaymentsInfo, AccountHistoryStorage, AccountHistory]] = {
+    BaseFeeBlockValidator() +: super.historyBlockValidators(params)
```

Comment:
let's switch the order: first validators from `super` then BaseFeeBlockValidator